build-plugin/src/main/kotlin/thunderbird.quality.kover.gradle.kts

```
@@ -0,0 +1,19 @@
+plugins {
+    id("org.jetbrains.kotlinx.kover")
+}
+
+kover {
+    reports {
+        filters {
+            excludes {
+                annotatedBy("androidx.compose.ui.tooling.preview.Preview")
```

We might need to exclude other Preview annotations as well:

```suggestion
                annotatedBy(
                    "androidx.compose.ui.tooling.preview.Preview",
                    "androidx.compose.ui.tooling.preview.PreviewLightDark",
                    "app.k9mail.core.ui.compose.designsystem.PreviewLightDarkLandscape",
                )
```

Prehabs, we could use a wildcard to include all `Preview*` annotations?